numcosmo/galaxy/nc_galaxy_sd_shape_gauss.c

```diff
@@ -437,6 +455,8 @@ _nc_galaxy_sd_shape_gauss_prepare_data_array (NcGalaxySDShape *gsds, NcmMSet *ms
                                                          &ldata_i->optzs);
   }
 
+  self->std_shape = nc_galaxy_sd_shape_gauss_std_shape_from_sigma (SIGMA);
```

Comment:
It maybe a good idea to have this happening in a separate function that would call model_update to check whether SIGMA was modified, updating self->std_shape when it was. *All* functions depending on self->std_shape needs to call this updating functions before accessing self->std_shape.